pkg/ctl/delete/iamserviceaccount.go

```diff
@@ -23,7 +23,7 @@ func deleteIAMServiceAccountCmd(cmd *cmdutils.Cmd) {
 
 	var onlyMissing bool
 
-	cmd.SetDescription("iamserviceaccount", "Create an iamserviceaccount - AWS IAM role bound to a Kubernetes service account", "")
+	cmd.SetDescription("iamserviceaccount", "Delete an iamserviceaccount", "")
```

Comment:
Can you change `iamserviceaccount` for "IAM service account" which is more readable?

Reply:
@martina-if I just pushed and committed the suggested edit. Let me know if you spot anything else.